_sections/00-Introduction.md

```diff
@@ -2,8 +2,15 @@
 title: Introduction
 ---
 
-The ODK XForms specification is a subset of the far larger [XForms 1.0 specification](http://www.w3.org/TR/xforms/). It contains a few additional features not found in the XForms specification.
+The ODK XForms specification is a subset of the far larger [W3C XForms 1.0 specification](http://www.w3.org/TR/xforms/) used by tools in the [Open Data Kit](https://opendatakit.org) ecosystem. It contains a few additional features not found in the W3C XForms specification.
 
-This specification continues to evolve. On the web you will often see references to _JavaRosa_ and _OpenRosa_ when the spec is discussed. OpenRosa was the name of the initial specification that formed the basis of this document. JavaRosa is a library that implements the XForms part of the OpenRosa specification.
+The purpose of this specification is to provide a common form description standard that many different kinds of compatible tools can be based on. Using a single, shared form description standard has the following advantages:
+1. Users in the ODK ecosystem can mix and match tools and reassess which they use based on their changing needs. In particular, they don't get locked in to tools that may become deprecated or for which an attractive replacement become available.
+1. Tool implementors in the ODK ecosystem can benefit from feedback from a broad range of collaborators when designing new core functionality.
+1. Tool implementors in the ODK ecosystem can share core implementations.
+
+This document is intended primarily for developers who build form processing engines or software form builders. Most organizations who use tools in the ODK ecosystem for data collection will prefer to create forms using the [XLSForm standard](http://xlsform.org/) or a graphical form builder.
+
+A version of this specification was initially developed by the [OpenRosa Consortium](https://bitbucket.org/javarosa/javarosa/wiki/OpenRosaAPI). [JavaRosa](https://bitbucket.org/javarosa/javarosa/wiki/Home) is a Java library developed by the consortium as a J2ME app that implements this specification.
```

Comment:
This only captures a fraction of what the OpenRosa group defined so I'm not sure it's a good idea to get rid of it quite yet. I don't believe any of the APIs are documented anywhere else.